client/payment-details/summary/test/index.test.tsx

```diff
@@ -461,390 +358,461 @@ describe( 'PaymentDetailsSummary', () => {
+		screen.getByText(
+			/The cardholder claims this is an unauthorized transaction/,
+			{ ignore: '.a11y-speak-region' }
+		);
+
+		// Don't render the staged evidence message
+		expect(
+			screen.queryByText( /You initiated a challenge to this dispute/, {
+				ignore: '.a11y-speak-region',
+			} )
+		).toBeNull();
+
+		// Dispute Summary Row
+		expect(
+			screen.getByText( /Dispute Amount/i ).nextSibling
+		).toHaveTextContent( /\$20.00/ );
+		expect(
+			screen.getByText( /Disputed On/i ).nextSibling
+		).toHaveTextContent( /Aug 30, 2023/ );
+		expect( screen.getByText( /Reason/i ).nextSibling ).toHaveTextContent(
+			/Transaction unauthorized/
+		);
+		expect(
+			screen.getByText( /Respond By/i ).nextSibling
+		).toHaveTextContent( /Sep 9, 2023/ );
+
+		// Steps to resolve
+		screen.getByText( /Steps to resolve/i );
+		screen.getByRole( 'link', {
+			name: /Email the customer/i,
+		} );
+		screen.getByRole( 'link', {
+			name: /guidance on dispute withdrawal/i,
 		} );
 
-		afterEach( () => {
-			global.wcpaySettings.featureFlags.isDisputeOnTransactionPageEnabled = false;
+		// Actions
+		screen.getByRole( 'button', {
+			name: /Challenge dispute/,
 		} );
+		screen.getByRole( 'button', {
+			name: /Accept dispute/,
+		} );
+	} );
 
-		test( 'renders the information of a disputed charge', () => {
-			const charge = getBaseCharge();
-			charge.disputed = true;
-			charge.dispute = getBaseDispute();
-			charge.dispute.status = 'needs_response';
+	test( 'renders the information of a disputed charge when the store/charge currency differ', () => {
+		// True when multi-currency is enabled.
+		global.wcpaySettings.shouldUseExplicitPrice = true;
 
-			renderCharge( charge );
+		// In this case, charge currency is JPY, but store currency is NOK.
+		const charge = getBaseCharge();
+		charge.currency = 'jpy';
+		charge.amount = 10000;
+		charge.balance_transaction = {
+			amount: 72581,
+			currency: 'nok',
+			reporting_category: 'charge',
+			fee: 4152,
+		};
+		charge.disputed = true;
+		charge.dispute = getBaseDispute();
+		charge.dispute.status = 'needs_response';
+		charge.dispute.amount = 10000;
+		charge.dispute.currency = 'jpy';
+		charge.dispute.balance_transactions = [
+			{
+				amount: -72581,
+				currency: 'nok',
+				fee: 15000,
+				reporting_category: 'dispute',
+			},
+		];
+		renderCharge( charge );
 
-			// Dispute Notice
-			screen.getByText(
-				/The cardholder claims this is an unauthorized transaction/,
-				{ ignore: '.a11y-speak-region' }
-			);
+		// Disputed amount should show the store (balance transaction) currency.
+		expect(
+			screen.getByText( /Dispute Amount/i ).nextSibling
+		).toHaveTextContent( /kr 725.81 NOK/i );
+	} );
 
-			// Don't render the staged evidence message
-			expect(
-				screen.queryByText(
-					/You initiated a challenge to this dispute/,
-					{ ignore: '.a11y-speak-region' }
-				)
-			).toBeNull();
+	test( 'renders the information of a dispute-reversal charge', () => {
```

Comment:
This is not a new test, simply moved down in this file to group dispute-related tests.